app/models/location.rb

```diff
@@ -311,6 +311,17 @@ def self.is_unknown?(name)
     false
   end
 
+  # Very abbreviated description of the location for shorter query titles.
+  # Just the location without locality, region, country
+  # Furthermore abbreviates the location if longer than 4 words
+  def title_display_name
+    if User.current_location_format == "scientific"
```

Comment:
Errr, I'm probably missing something!... but since you're only taking one term, does it matter whether the user's location format is scientific or not?  Wouldn't `name.split(", ").first` be the same either way?

Reply:
Oh, riiiiiight!